src/bindgen/index.md

```diff
@@ -48,16 +48,17 @@ Then add `bindgen` as a dependency and tell `Cargo` about our build script
 
 `Bindgen` works by reading a `C`-style header file using `libclang`, then
 it runs the preprocessor and creates bindings based on the output. Therefore, 
-before we can start our build script we'll need a dummy header which pulls in
-the `bzip2` library installed on your system.
+the build script needs a dummy header which pulls in the `bzip2` library
+installed on your system.
```

Comment:
If we're pulling in `bzip2`, should there be a "Requirements" note higher up saying that you should have `bzip2` installed somewhere? 

I'm not sure how Windows deals with finding include files, is there some equivalent of `/usr/include/` on a Windows machine? I feel like we've written a lot of this guide with the implicit assumption that you are on a unix-y machine, so I'm not sure how much will work out of the box for Windows users.